src/frontend/components/likedvideo/LikedVideo.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useVideoContext } from '../../context/index-context'
+import { SideMenuGuide, LikedVideoCard} from '../index-components'
+import { Link } from "react-router-dom"
+
+const LikedVideo = () => {
+  const { state } = useVideoContext()
+  const { likedData} = state  
+  return (
+    <div className="single-card-wrapper">
+      <SideMenuGuide />
+      <div className = "single-play-card">
+        {likedData.map((ele) => {
+            return <LikedVideoCard vInfo = {ele}/>
+        })}
```

Comment:
Whenever you map on something, try to name the parameter as what each element is. Eg., Here, likedVideo.